process-dir.js

```diff
@@ -184,6 +196,41 @@ async function replacePackage (dir, file, files) {
   await writeFile(pkgFile, JSON.stringify(pkg, null, 2))
 }
 
+function hasNodeEnvCondition (source) {
+  return /process.env.NODE_ENV\s*[!=]==\s*["']production["']/.test(source)
+}
+
+async function replaceNodeEnvConditions (dir, file, source) {
+  source = source.toString()
+  let prodCondition = /process.env.NODE_ENV\s*===\s*["']production["']/g
```

Comment:
Maybe we should also respect `process.env.NODE_ENV === "development"`?

Reply:
Yeap, we need it too

Reply:
Maybe someone uses not strict equality, i.e. `==` will add it too

Reply:
And maybe backticks should be considered?

Reply:
Added support for "development", not strict equality and strings in backticks